.github/workflows/build_tests.yml

```diff
@@ -0,0 +1,39 @@
+name: Run Build Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - dev
+  workflow_dispatch:
+
+jobs:
+  py_build_tests:
+    uses: neongeckocom/.github/.github/workflows/python_build_tests.yml@master
+    with:
+      test_manifest: true
+      manifest_ignored: "test/**,scripts/**,CHANGELOG.md,ui/**,locale/**,**.txt"
+  pip_audit:
+    strategy:
+      max-parallel: 2
+      matrix:
+        python-version: ["3.10"]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install skill
+        run: |
+          pip install .
+      - uses: pypa/gh-action-pip-audit@v1.0.7
```

Comment:
`pypa/gh-action-pip-audit@v1.0.8`

 pip-audit itself is pulling requests<x -> vuln with 1.07

Reply:
fixed